src/sparseml/exporters/transforms/matmul_to_matmulinteger_add_cast_mul.py

```diff
@@ -21,65 +21,61 @@
     INITIALIZER_MATCH,
     MatchResult,
     add_quantized_conv_matmul_add_ops,
-    delete_quant_node,
     get_quantization_params,
     get_structural_matches,
     optional_node,
 )
-from sparseml.onnx.utils import (
-    ONNXGraph,
-    get_init_by_name,
-    remove_node_and_params_from_graph,
-)
-
+from sparseml.onnx.utils import ONNXGraph
 
-_LOGGER = logging.getLogger(__name__)
 
 __all__ = ["MatMulToMatMulIntegerAddCastMul"]
 
 
+_LOGGER = logging.getLogger(__name__)
+
+
 class MatMulToMatMulIntegerAddCastMul(OnnxTransform):
     """
     A transform for converting a MatMul with kernel and bias into a
     quantized representation
 
     ```
-    | Starting with:
-    |                        weight (initializer)
-    |                            |
-    |                      QuantizeLinear
-    |                            |
-    |          INPUT     DequantizeLinear
-    |            |               |
-    |     DequantizeLinear   Transpose
-    |                  |      |
-    |                   MatMul    bias (initializer)
-    |                         |   |
-    |                          Add
-    |                          |
-    |                      QuantizeLinear (Optional)
-    |                          |
-    |                      DequantizeLinear (Optional)
-    |                          |
-    |                       OUTPUT
-    | We end up converting to:
-    |       INPUT
+    |     weight (initializer)
     |         |
-    |     MatMulInteger (with constant uint8 kernel)
+    |         Q
     |         |
-    |     Add (constant bias + zero point correction)
+    | input   Dq
+    |   |     |
+    |   Dq   Transpose
+    |     |   |
+    |     MatMul  bias (initializer)
+    |         |   |
+    |         Add
     |         |
-    |     Cast (INT32 -> FP32)
+    |     optional Q
     |         |
-    |     Mul (Rescale from bias scale)
-    |         |
-    |       OUTPUT
+    |     optional Dq
+    ```
+    (where `Q` is QuantizeLinear, and `Dq` is DequantizeLinear)
+    into
+    ```
+    |   input
+    |     |
+    | MatMulInteger (with constant uint8 kernel)
+    |     |
+    | Add (constant bias + zero point correction)
+    |     |
+    | Cast (INT32 -> FP32)
+    |     |
+    | Mul (Rescale from bias scale)
     ```
     """
 
     def transform(self, model: ModelProto) -> ModelProto:
+        graph = ONNXGraph(model)
+        count = 0
         matches = get_structural_matches(
-            ONNXGraph(model),
+            graph,
             op_type="MatMul",
             parent_ops=[
                 ["DequantizeLinear"],
```

Comment:
This should be `any_of("QuantizeLinear", "DequantizeLinear")`. also update docstring